dev-python/pgspecial/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <maintainer type="project">
+    <email>python@gentoo.org</email>
```

Comment:
Don't add random python ebuilds to be maintained by python project. You need to maintain it if you wish it to be in the tree (add yourself and proxy-maint here too)